smart-contracts/test/Unlock/UnlockProxy.js

```diff
@@ -36,12 +38,12 @@ contract('Unlock', function (accounts) {
 
     describe('should function after upgrade', function () {
       beforeEach(async function () {
-        await this.project.upgradeProxy(this.proxy, UnlockTestV2, { initMethod: 'initializeV2', initArgs: [], initFrom: unlockOwner })
+        await shell.exec('zos add UnlockTestV2:Unlock --skip-compile')
+        await this.project.upgradeProxy(this.proxy.address, UnlockTestV2, { Unlock, initMethod: 'initializeV2', initArgs: [] })
```

Comment:
So now I am confused. Are we using the `zos` cli or the library?

Reply:
Both...( although that reminds me to double check that we're still using  `zos-lib` ;)). The `zos` package for the cli contains the `TestHelper` which is meant to be used in our tests. I've also invoked the cli via `shell.exec`, but even without using `shell.exec` we'd need the `zos`  package to access the `TestHelper`.

Reply:
Edit: We're not using the zos-lib anymore in our tests, but we use its `Initializable.sol` in our contracts.